lib/helpers.js

```diff
@@ -39,19 +39,11 @@ var helpers = module.exports = {
     // arrays into objects in the output.
     // We now check if an array is only in the child or parent, return that.
     if(_.isArray(b) && !a){
-      if(_.every(b, _.isObject)){
-        return _.merge(a,b);
-      } else {
-        return b;
-      }
+      return b;
     }
 
     if(_.isArray(a) && !b){
-      if(_.every(a, _.isObject)){
-        return _.merge(a,b);
-      } else {
-        return a;
-      }
+      return a;
```

Comment:
@paulmiard think it's related to a comment I left in https://github.com/prolificinteractive/ballad/pull/4. The condition didn't make much sense.